marklogic-data-hub/src/test/java/com/marklogic/hub/security/DataHubOperatorTest.java

```diff
@@ -32,4 +40,25 @@ void readSchemas() {
                 "permissions on them. Since they may have no other read permissions, the operator requires this so that " +
                 "it can read entity models.");
     }
+
+    @Test
+    void testQBE(){
+        writeFinalJsonDoc("doc1.json", "{\"itemPrice\": 10}");
+        writeFinalJsonDoc("doc2.json", "{\"itemPrice\": 5}");
+
+        QueryManager queryMgr = runAsDataHubOperator().getFinalClient().newQueryManager();
+        String rawJSONQuery = "{ \"$query\": {\n" +
+            "    \"$and\": [\n" +
+            "        {\"itemPrice\" : { \"$ge\": 6 } },\n" +
+            "        {\"itemPrice\" : { \"$le\": 12 } } ],\n" +
+            "    \"$filtered\": true\n" +
+            "}}";
+        StringHandle rawHandle = new StringHandle(rawJSONQuery).withFormat(Format.JSON);
+        RawQueryByExampleDefinition querydef = queryMgr.newRawQueryByExampleDefinition(rawHandle);
+        SearchHandle resultsHandle = queryMgr.search(querydef, new SearchHandle());
+        assertEquals(1, resultsHandle.getTotalResults());
```

Comment:
Can you add an assertion message stating that this works because the user has the eval-search-string privilege?